lib/htmlValidator.js

```diff
@@ -170,10 +170,12 @@ module.exports = function(app, callback) {
     next();
   });
 
-  promise.then(function(result) {
+  Promise.all([promise]).then((result) => {
+    if (!params.enableValidator) {
+      console.error(('HTML Validator disabled. Check to make sure you have HTML Validation enabled. Continuing without HTML validation...').red);
```

Comment:
do a quick audit of all the logs here and change the capitalization from "HTML Validator" to "HTML validator"

as to this message specifically, go with: "⚠️  HTML validator disabled. Continuing without HTML validation..."

also change it to a warning (in yellow) and do not display the warning at all unless it is dev mode (the validation should always be disabled in prod mode)